<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainmenu</class>
 <widget class="QWidget" name="mainmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>677</y>
     <width>277</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="adinfo">
   <property name="geometry">
    <rect>
     <x>792</x>
     <y>310</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>541</y>
     <width>249</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Changetar">
   <property name="geometry">
    <rect>
     <x>604</x>
     <y>541</y>
     <width>249</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="change_pass">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>388</y>
     <width>249</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Changetar_2">
   <property name="geometry">
    <rect>
     <x>604</x>
     <y>388</y>
     <width>249</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Newhouse">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>131</y>
     <width>217</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="Homes">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>521</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-color: rgb(92, 53, 102);
selection-background-color: rgb(173, 127, 168);
color: rgb(238, 238, 236);
background-color: rgba(173, 127, 168,0.1);</string>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_house">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>188</y>
     <width>217</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="delacc">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>388</y>
     <width>249</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="add_room">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>131</y>
     <width>217</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="info_house">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>243</y>
     <width>217</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dele_room">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>187</y>
     <width>217</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Change_h">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>299</y>
     <width>217</width>
     <height>47</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Countet_butt">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>464</y>
     <width>249</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="chang_room">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>244</y>
     <width>217</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Devais_butt">
   <property name="geometry">
    <rect>
     <x>604</x>
     <y>464</y>
     <width>249</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Sencor_butt">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>464</y>
     <width>249</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bank">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>541</y>
     <width>249</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>755</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pich.qrc">:/picha/Pichures/MenuScreenUser.png</pixmap>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>Newhouse</zorder>
  <zorder>pushButton</zorder>
  <zorder>adinfo</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>Changetar</zorder>
  <zorder>change_pass</zorder>
  <zorder>Changetar_2</zorder>
  <zorder>Homes</zorder>
  <zorder>delete_house</zorder>
  <zorder>delacc</zorder>
  <zorder>add_room</zorder>
  <zorder>info_house</zorder>
  <zorder>dele_room</zorder>
  <zorder>Change_h</zorder>
  <zorder>Countet_butt</zorder>
  <zorder>chang_room</zorder>
  <zorder>Devais_butt</zorder>
  <zorder>Sencor_butt</zorder>
  <zorder>bank</zorder>
 </widget>
 <resources>
  <include location="pich.qrc"/>
 </resources>
 <connections/>
</ui>
